src/styles/styles.js

```diff
@@ -0,0 +1,34 @@
+export const styles = {
+  colors: {
+    gray1: "rgb(45, 45, 45)",
+    gray2: "rgb(70, 70, 70)",
+    blue1: "rgb(0, 136, 235)",
+    tBlue: "rgba(0, 136, 235, 0.75)",
+  },
+  sizes: {
+    thickness: {
+      thick1: 1,
+      thick2: 2,
+      thick3: 4,
+      thick4: 5,
+      thick5: 10,
```

Comment:
I think jumping from 5 to 5 will be problematic. Let's jump 2 from to and define up to 20